src/Helpers/SearchClientStub.php

```diff
@@ -0,0 +1,251 @@
+<?php
+declare(strict_types=1);
+
+namespace Eonx\TestUtils\Helpers;
+
+use Eonx\TestUtils\Helpers\Interfaces\ClientStubInterface;
+use LoyaltyCorp\Search\DataTransferObjects\ClusterHealth;
+
+final class SearchClientStub implements ClientStubInterface
```

Comment:
I think we need to add a deliverable to the "stubs" ticket to modify this stub to follow the new standard (that will be added to this repo)

Reply:
And any other stubs that might be needed as part of this pr

Reply:
I could not find what ticket you are talking about. should this be part of deliverable when the test-utils are implemented in `subs`?